src/ui/public/vis/response_handlers/tabify.js

```diff
@@ -32,7 +33,8 @@ const TabifyResponseHandlerProvider = function (Private) {
         const tableGroup = aggResponse.tabify(vis.getAggConfig(), response, {
           canSplit: true,
           asAggConfigResults: _.get(vis, 'type.responseHandlerConfig.asAggConfigResults', false),
-          isHierarchical: vis.isHierarchical()
+          isHierarchical: vis.isHierarchical(),
+          timeRange: getTime(vis.indexPattern, vis.filters.timeRange).range
```

Comment:
I think the same will also happen here if you don't have a time field in your index pattern